letsencrypt/plugins/standalone/authenticator.py

```diff
@@ -301,10 +305,21 @@ def already_listening(self, port):  # pylint: disable=no-self-use
         :param int port: The TCP port in question.
         :returns: True or False."""
 
-        listeners = [conn.pid for conn in psutil.net_connections()
+        try:
+            net_connections = psutil.net_connections()
+        except psutil.AccessDenied as error:
+            logger.info("Access denied when trying to list network "
+            "connections: %s. Are you root?", error)
```

Comment:
This is missing indent (should match opening quote from line above), but I don't know why pylint didn't catch it...